### ISSUE TITLE
iOS: Project template has no momentum scrolling

### ISSUE THREAD

**Issue by leonderijke** [kabisa/maji#204]
Because of the use of `overflow: auto` on the `main` element iOS will not use the "momentum scrolling".

**Comment by pascalw** [comment]
If I remember correctly iOS will only use momentum scrolling if you put `-webkit-overflow-scrolling: touch` on the scrolling element.

Putting this on the main element might be nice for a demo, but if you actually want momentum scrolling you'll still need to think were to add this in your app. Or am I missing something?

**Comment by leonderijke** [comment]
Yes, that's correct. However, momentum scrolling can be a drain on battery life and used to be known to cause tricky side effects.

I'm suggesting that maybe we should add it to the project template, but we should test it really thoroughly.
